torch/optim/swa_utils.py

```diff
@@ -177,7 +177,7 @@ class SWALR(_LRScheduler):
         swa_lrs (float or list): the learning rate value for all param groups
             together or separately for each group.
         annealing_epochs (int): number of epochs in the annealing phase 
-            (default: 10)
+            (default: 0)
```

Comment:
To avoid BC breaking change, let's just keep the default as it was.

```suggestion
            (default: 10)
```

Reply:
Got it. Thanks for the review and fix!